frontend/src/Troop/TroopInformation.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { View, Image, Text } from 'react-native';
+import commonStyles from '../common/styles';
+import troopAvatar from '../../assets/troop/troop-avatar.jpg';
+
+function TroopInformation() {
+  return (
+    <View style={{ marginRight: 24 }}>
+      <Image style={{ width: 100, height: 100, borderRadius: 10 }} source={troopAvatar} />
+      <Text style={commonStyles.textStyle}>Attack: 35</Text>
+      <Text style={commonStyles.textStyle}>Defence: 35</Text>
+      <Text style={commonStyles.textStyle}>Sustenance: 23</Text>
```

Comment:
added icons for troops and changed the level text format, but I cannot find icons for sustenance on the assets folder on google drive, so I find one by myself, hopefully it will work.